__tests__/app.test.js

```diff
@@ -628,4 +628,72 @@ describe('demo routes', () => {
     });
   });
 
+  it('/DELETE with wrong user, verify error message', async() => {
+    const agent = request.agent(app);
+    const agent2 = request.agent(app);
+    
+    const user = await UserService.create({
```

Comment:
I would suggest, to make these files more compact if you have the time to refactor, that you could create users and log them in in your before each.  So that you don't have repeats.  However, this way works just as well :) Nice work, friends!